### ISSUE TITLE
Issue: Playback looped after using the Slip tool

### ISSUE THREAD

**Issue by Efenstor** [jliljebl/flowblade#543]
Flowblade version (Help->About): 1.16.0
MLT version (Help->Runtime Environment): 6.8.0
Your distribution (Ubuntu, Debian, Mint etc.): Debian Stretch

After shifting the clip backwards using the Slip tool (not switching to Insert or Overwrite aftewards) the playback is endlessly looped for the amount of time shifted.

**Comment by jliljebl** [comment]
Confirmed. This was working, so there has been regression sometime.

**Comment by jliljebl** [comment]
Now I'm not seeing anything wrong. 

So when in any of the trim modes the play back is *looping on purpose* and to get non-looping playback one needs to exit *active* trim mode by clicking on empty. Because trim view is displayed as a hidden clip on top of all video tracks *it is impossible to offer non-looping and free playback* when trims are active.

I though I saw some shift in the position of playback, but I have not been able to reproduce that. I will:
1) Try get it reproducing again on the shift of looping, but maybe I just saw ghosts.
2) Consider changing the Play button icon to image describing looping when in active trim modes and looping will happen.
